clusterclient.go

```diff
@@ -0,0 +1,120 @@
+package kafka
+
+import (
+	"context"
+	"fmt"
+)
+
+// ClusterClient provides a high-level API for miscellaneous (i.e. non consume or produce) Kafka cluster operations
+type ClusterClient struct {
+	brokers []string
+	dialer  *Dialer
+}
+
+// NewClusterClient creates and returns a *ClusterClient with a []string of bootstrap
+// brokers for connecting to the cluster and the DefaultDialer
+func NewClusterClient(brokers []string) *ClusterClient {
+	return &ClusterClient{
+		brokers: brokers,
+		dialer:  DefaultDialer,
+	}
+}
+
+// NewClusterClient creates and returns a *ClusterClient with a []string of bootstrap
+// brokers for connecting to the cluster and a user supplied Dialer
+func NewClusterClientWith(brokers []string, d *Dialer) *ClusterClient {
+	return &ClusterClient{
+		brokers: brokers,
+		dialer:  d,
+	}
+}
+
+// ConsumerOffsets returns a map[int]int64 of partition to committed offset for a consumer group id and topic
+func (cc *ClusterClient) ConsumerOffsets(ctx context.Context, groupId string, topic string) (map[int]int64, error) {
```

Comment:
Should it be `topic string, groupId string` ? 😅  or should we have an API that helps programs not mistakenly invert the arguments maybe?

Reply:
done